content/english/linux-basics/activity-6.md

```diff
@@ -25,21 +26,20 @@ A `char` is any letter, symbol, or number. Examples include `!`, `h`, `2`, etc.
 | -n | To return matched lines and their line numbers. |
 | -R | Search all files in current directory. |
 
-*Let's practice this command to find Nuvi's secret-message.txt! Let's try the following command:*
+*Let's practice this command to find my secret-message.txt! Let's try the following command:*
 
 ```
 cd
 grep -R "Do. Or do not. There is no try."
 ```
 
-![grep command](../images/Act6.1.PNG?classes=border,shadow)
+![grep command](../images/Act6.1.png?classes=border,shadow)
 
 --------------------------
 
 ### Why does the secret-message file look like that?
 
-*Remember that file we saw with the strange name? It was called `malware.sh`. We should go find it again and take a closer look 
-at it.*
+*Remember that file we saw with the strange name? It was called `malware.sh`. We should go find it again and take a closer look at it. I think this was left by the hackers!*
 
 `find` is a command used to search for a file with certain parameters within a directory or region of files. It can be used in multiple ways including:
 
```

Comment:
- Searching for a file with a specific file name

Reply:
- Searching for all fixes with a specific file extension (for example, text files, ending with .txt)

Reply:
Updated the bullet points.